src/wash/common/Modal.tsx

```diff
@@ -34,9 +35,7 @@ export function Modal({
           leaveTo={'opacity-0'}
         >
           <div
-            className={
-              'fixed inset-0 bg-[#1B1036CC] backdrop-blur transition-opacity'
-            }
+            className={`fixed inset-0 bg-[${colors.violet[100]}CC] backdrop-blur transition-opacity`}
```

Comment:
I would suggest to move this to style, as the css property like this might cause errors on production. As we want to remove classname anyway, this might be better to avoid any potential issue!
```suggestion
            style={{
              position: 'fixed',
              inset: 0,
              background: `${colors.violet[100]}CC`,
              backdropFilter: 'blur',
              transition: 'opacity 300ms ease-in-out',
            }}
```